magento/magento2
# Graphql events.xml is added for quote submit succes to trigger sales …

…email





### Description (*)
Sales email is not sent after order placement through graphql requests. The fix includes event that triggers sales emails after order placement.


### Related Pull Requests


### Fixed Issues (if relevant)
#28124  is fixed with this PR.

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)


1. set `sales_email/order/enabled` to yes
1. Follow [official graphql order placement document](https://devdocs.magento.com/guides/v2.3/graphql/tutorials/checkout/checkout-customer.html )to place order
2. Observe order confirmation email is received

### Questions or comments
I haven't seen any test related to this feature as it's used with soap and rest events.xml. I am happy to add related test if required.


### Contribution checklist (*)
